thread-ids.lisp

```diff
@@ -24,10 +24,8 @@
        *thread-ids*)))
 
   (defun save-current-thread-id ()
```

Comment:
This should be declaimed notinline so the later redefinition is guaranteed to be seen.